components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/APIDefinitionFromSwagger20.java

```diff
@@ -281,51 +281,71 @@ public String createAPIDefinition(API api) throws APIManagementException {
         String endpoints = environment.getApiGatewayEndpoint();
         String[] endpointsSet = endpoints.split(",");
         Set<URITemplate> uriTemplates = api.getUriTemplates();
+        Set scopes = api.getScopes();
```

Comment:
The code has been already fixed in the mentioned way in PR[1]. Thanks for the comments
[1] - https://github.com/wso2/carbon-apimgt/pull/282/files